src/suites/cts/validation/setBlendColor.spec.ts

```diff
@@ -0,0 +1,45 @@
+export const description = `
+setBlendColor validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+// TODO: Move beginRenderPass to a Fixture class.
+export class F extends ValidationTest {
+  beginRenderPass(commandEncoder: GPUCommandEncoder): GPURenderPassEncoder {
+    const attachmentTexture = this.device.createTexture({
+      format: 'rgba8unorm',
+      size: { width: 16, height: 16, depth: 1 },
+      usage: GPUTextureUsage.OUTPUT_ATTACHMENT,
+    });
+
+    return commandEncoder.beginRenderPass({
+      colorAttachments: [
+        {
+          attachment: attachmentTexture.createView(),
+          loadValue: { r: 1.0, g: 0.0, b: 0.0, a: 1.0 },
+        },
+      ],
+    });
+  }
+}
+
+export const g = new TestGroup(F);
+
+g.test('basic use of setBlendColor', t => {
+  const commandEncoder = t.device.createCommandEncoder();
+  const renderPass = t.beginRenderPass(commandEncoder);
+  renderPass.setBlendColor({ r: 0, g: 0, b: 0, a: 0 });
+  renderPass.endPass();
+  commandEncoder.finish();
+});
+
+g.test('setBlendColor allows any number value', t => {
+  const commandEncoder = t.device.createCommandEncoder();
+  const renderPass = t.beginRenderPass(commandEncoder);
+  renderPass.setBlendColor({ r: -1, g: 42, b: -0, a: 0 });
```

Comment:
GPUColor use double values. See https://gpuweb.github.io/gpuweb/#dictdef-gpucolordict
Since the double type is a floating point numeric type that corresponds to the set of **finite** double-precision 64 bit IEEE 754 floating point numbers, it makes sense to me to test only `MIN_SAFE_INTEGER` and `MAX_SAFE_INTEGER`.
WDYT?